R/as_OpenSpecy.R

```diff
@@ -260,8 +260,16 @@ as_OpenSpecy.default <- function(x, spectra,
   obj <- structure(list(), class = c("list", "OpenSpecy"))
 
   obj$wavenumber <- x
+  
+  if(!(identical(order(x), 1:length(x)) | identical(order(x), length(x):1))){
+      warning("Wavenumbers should be a continuous sequence for all OpenSpecy functions to run smoothly. Consider sorting your wavenumbers and spectral intensities before turning into an OpenSpecy object.")
+  }
```

Comment:
Since sorting doesn't require any additional information and I cannot think of a scenario in which sorting the data might not be desired or problematic, I suggest to sort automatically once the spectra is converted into an OpenSpecy object. Then, we can safely assume that every OpenSpecy object has sorted wavenumbers.

Reply:
True, it is also relatively cheap. Just committed. https://github.com/wincowgerDEV/OpenSpecy-package/pull/129/commits/b46e91df39a90137254eae2a82c969c3b4256253